components/Banner/src/MDCBannerView.m

```diff
@@ -60,9 +60,9 @@ @interface MDCBannerView ()
 // Buttons constraints
 @property(nonatomic, readwrite, strong) NSLayoutConstraint *buttonContainerConstraintLeading;
 @property(nonatomic, readwrite, strong)
-    NSLayoutConstraint *buttonContainerConstraintLeadingWithTextLabel;
+    NSLayoutConstraint *buttonContainerConstraintWidthWithLeadingButton;
```

Comment:
_Tip:_ buttonContainerWidthConstraint
Also documentating these properties would help our future selves understand what they are for.

Reply:
I felt `..WithLeadingButton` would be more explicit for now when it is a property inside the implementation file. How do you feel if we keep it for now? I am thinking there might be more width constraints to be added later on so I added this suffix.

Reply:
Sounds good, I think docs would make it more clear what this constraint is responsible for.

Reply:
Sounds like a general doc would be good for these constraints. I will address it as a separate issue :)